### ISSUE TITLE
How to train the net without using pretrained model?

### ISSUE THREAD

**Issue by daoanhvu** [nilboy/tensorflow-yolo#17]
Hi 
Could you please guide me how to train the net without using pretrained model?

I faced with this error when trying train without pretrained file:
NotFoundError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to find any matching files for 
	 [[Node: save/RestoreV2_8 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_8/tensor_names, save/RestoreV2_8/shape_and_slices)]]

Thanks

**Comment by nilboy** [comment]
in file yolo/solver/yolo_solver.py, 
comment  line 79: saver1.restore(sess, self.pretrain_path); 
And use small learning rate

**Comment by daoanhvu** [comment]
Thanks, but where can I set learning rate?
Thank you!
